assets-styleguide/js/styleguide.js

```diff
@@ -114,6 +121,17 @@ $('.sidenav').on('click', 'a', function(e) {
       } else if (window.location.hash !== newHash) {
         window.location.hash = newHash;
       }
+    },
+    done: function () {
```

Comment:
I did a bit of digging about accessible smooth scrolls and tested a couple with the screen reader. At most, focusing after the animation would cause a minor inconvenience, and even that is a bit of a stretch. 

As long as its clear, that its a link targeting another section on the page, and you move focus to the new area after the animation, it should be good. 

I don't think disabling smooth scrolling for keyboard navigation is necessary.